crates/notion-core/src/tool/npm.rs

```diff
@@ -22,13 +22,21 @@ impl Tool for Npm {
             throw!(ErrorDetails::NoGlobalInstalls);
         }
 
-        if let Some(ref platform) = session.current_platform()? {
+        // if we're in a pinned project, use npm from that platform
+        if let Some(ref platform) = session.project_platform()? {
             let image = platform.checkout(session)?;
             Ok(Self::from_components(
                 OsStr::new("npm"),
                 args,
                 &image.path()?,
             ))
+        } else if let Some(user_tool) = session.get_user_tool(OsStr::new("npm"))? {
```

Comment:
Will this result in a failure if the user is outside of a project and has only done `notion install node`, but not `notion install npm`? We should default to the bundled `npm` in that case, so if it does cause an error, we should also include an `else if let Some(ref platform) = session.user_platform()?` block that catches that case.

Reply:
Ah, right - I should remove this for now, since installing `npm` is not implemented at this point